indico/modules/events/timetable/client/js/TimetableManageModal.tsx

```diff
@@ -77,7 +77,8 @@ interface DraftEntry {
   session_id?: number;
   code?: string;
   board_number?: string;
-  id?: number;
+  objId?: number;
+  id?: string;
```

Comment:
Do we actually still need the `id`? I see in some places we now check the draftEntry's `objId`